api/src/controllers/users.js

```diff
@@ -127,7 +127,7 @@ module.exports = {
   },
   create: (req, res) => {
     return auth
-      .check(req, 'can_create_users')
+      .check(req, ['can_edit', 'can_create_users'])
```

Comment:
Change to `hasAllPermissions` (as above).

Reply:
`check` was refactored